src/lib/sender.js

```diff
@@ -188,11 +188,22 @@ function createSender (opts) {
       })
   }
 
+  /**
+   * Disconnect from the ledger and stop listening for events.
+   *
+   * @return {Promise.<null>} Resolves when the sender is disconnected.
+   */
+  function stopListening () {
+    client.removeListener('outgoing_fulfill', fulfillmentListener)
```

Comment:
This will throw an error because `fulfillmentListener` is not defined in this scope.

Note each listener is removed [when the fulfillment is received](https://github.com/vhpoet/js-ilp/blob/02edb991fb259969196b1045013c693c59c62400/src/lib/sender.js#L181).